ci.bash

```diff
@@ -167,7 +167,13 @@ function test::report
 function test::capture
 {
     if which juju-crashdump; then
-        juju-crashdump -s -a debug-layer -a config -m "$JUJU_CONTROLLER:$JUJU_MODEL"
+        juju-crashdump \
+            -s                   # small crashdump by skipping /var/lib/juju \
+            -a debug-layer       # included debug-layer addon \
+            -a config            # included config addon \
+            -j snap.kube*        # included logs from all kube* daemons \
+            -j snap.cdk-addons*  # included logs from cdk-addons*  \
+            -m "$JUJU_CONTROLLER:$JUJU_MODEL"
```

Comment:
I'm pretty sure anything after the `#` is not interpreted, including the end of line `\`.

```suggestion
        juju-crashdump \
            -s \                   # small crashdump by skipping /var/lib/juju
            -a debug-layer \       # included debug-layer addon
            -a config \            # included config addon
            -j snap.kube* \        # included logs from all kube* daemons
            -j snap.cdk-addons* \  # included logs from cdk-addons*
            -m "$JUJU_CONTROLLER:$JUJU_MODEL"
```